file/read_write_util.h

```diff
@@ -26,4 +26,9 @@ extern Status NewWritableFile(Env* env, const std::string& fname,
 bool ReadOneLine(std::istringstream* iss, SequentialFile* seq_file,
                  std::string* output, bool* has_data, Status* result);
 
+#ifndef NDEBUG
+bool IsFileSectorAligned(const size_t off, size_t sector_size) {
```

Comment:
I think we should mark this inline if we're going to put the implementation in the header file (alternatively, the definition could be moved to the .cc).

Reply:
Yes, I noticed it too. I moved it to .cc file now.